src/tests/components/App.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../../components/App";
+
+describe("App", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("renders correctly", () => {
```

Comment:
This test is failing due to out of date snapshots. Always run your tests before submitting and check they are all passing, updating the snapshots if needed!